.github/workflows/dockerhub.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
       - name: Determine master git tag
-        if: github.ref == 'refs/heads/carma-master' || github.ref == 'refs/heads/master' || contains(github.ref, 'tags/carma-system-')
+        if: github.ref == 'refs/heads/carma-master' || github.ref == 'refs/heads/master' || contains(github.ref, 'tags/')
```

Comment:
It is probably low chance, but probably better to look for refs/tags/
instead of just tags/ as some coincidental words might trigger it

Reply:
Good catch